CHANGELOG.md

```diff
@@ -100,6 +100,10 @@ See https://docs.voltocms.com/upgrade-guide/ for more information. @sneridagh
 
 ## 16.0.0-alpha.17 (2022-07-25)
 
+### Feature
+
+- Adding `Undo controlpanel` in the controlpanels, using it undone is possible on the transactions. @MdSahil-oss
```

Comment:
"Added the `Undo controlpanel` to the controlpanels which can be used to undo transactions."